tst/EnergyPlus/api/test_SysExit.py

```diff
@@ -0,0 +1,116 @@
+#
+# (3) Neither the name of the University of California, Lawrence Berkeley
+#     National Laboratory, the University of Illinois, U.S. Dept. of Energy nor
+#     the names of its contributors may be used to endorse or promote products
+#     derived from this software without specific prior written permission.
+#
+# (4) Use of EnergyPlus(TM) Name. If Licensee (i) distributes the software in
+#     stand-alone form without changes from the version obtained under this
+#     License, or (ii) Licensee makes a reference solely to the software
+#     portion of its product, Licensee must refer to the software as
+#     "EnergyPlus version X" software, where "X" is the version number Licensee
+#     obtained under this License and may not use a different name for the
+#     software. Except as specifically required in this Section (4), Licensee
+#     shall not use in a company name, a product name, in advertising,
+#     publicity, or other promotional activities any name, trade name,
+#     trademark, logo, or other designation of "EnergyPlus", "E+", "e+" or
+#     confusingly similar designation, without the U.S. Department of Energy's
+#     prior written consent.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+import shlex
+import tempfile
+import unittest
+
+from pyenergyplus.api import EnergyPlusAPI
+
+
+class TestAPISysExit(unittest.TestCase):
+    """
+    unittest on EnergyPlusAPI to see the exit and flushing to eplusout.err
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        A class method called before tests in an individual class are run
+        """
+        cls.api = EnergyPlusAPI()
+
+    def setUp(self):
+        """
+        Prepare each test fixture, with a new state
+        """
+        self.state = self.api.state_manager.new_state()
+
+    def test_wrong_file_sys_exit(self):
+        """
+        Test for #8483 - system exit called when energyplus can't run
+        """
+        with tempfile.TemporaryDirectory() as tempdir:
+            cmd_args = shlex.split(f'-d {tempdir} -w empty.epw doesntexist.idf')
+            return_code = self.api.runtime.run_energyplus(self.state, cmd_args)
+            self.assertEqual(return_code, 1)
```

Comment:
at ba0a4685aea9dbba7838473eda53ae67985dbf94, we see that system exit is called

```
test 89
    Start 89: API.test_SysExit

89: Test command: /home/julien/Virtualenvs/py38/bin/python3.8 "-m" "unittest" "/home/julien/Software/Others/EnergyPlus-build-release/Products/test_SysExit.py"
89: Test timeout computed to be: 10000000
89: ERROR: Could not find input data file: /home/julien/Software/Others/EnergyPlus-build-release/Products/doesntexist.idf.
89: Type 'energyplus --help' for usage.
1/1 Test #89: API.test_SysExit .................***Failed    0.10 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   0.19 sec

The following tests FAILED:
	 89 - API.test_SysExit (Failed)
```

at c4d92e9eb2558217a1693d2f3b3680e85c12ce6d it passes.